src/content/jetbrains/index.html

```diff
@@ -23,7 +23,7 @@
 
 	<h4>Activating theme</h4>
     <ol>
-        <li>Create a folder called <code>colors</code> inside <code>~/Library/Preferences/<b>{NameOfIDE}</b></code>.</li>
+        <li>Create a folder called <code>colors</code> inside <code>~/<b>.{NameOfIDE}</b>/config/colors/</code>.</li>
         <li>Copy the <code>./jetbrains/Dracula.icls</code> file to the <code>~/Library/Preferences/<b>{NameOfIDE}</b>/colors</code> directory on your local machine.</li>
```

Comment:
We need change this line too, so the user knows that it needs to be copied to the right folder